pom.xml

```diff
@@ -75,14 +75,14 @@
   </distributionManagement>
 
   <repositories>
-    <repository>
+    <!--repository>
```

Comment:
If bintray is used just as a central mirror, everything should be fine.

It seems though, that the swagger plugin depends on stuff that are exclusive to bintray, so we have to use it and provide an additional mirror for that in nexus.